apollo-client/src/main/java/com/ctrip/framework/apollo/internals/ConfigMonitorInitializer.java

```diff
@@ -102,10 +103,14 @@ private static void initializeMetricsEventListener() {
 
   private static void initializeMetricsExporter(
   ) {
+    if (StringUtils.isEmpty(m_configUtil.getMonitorExternalType()) || "NONE".equals(m_configUtil.
```

Comment:
Use isBlank instead of isEmpty

---

"NONE" is a hard code, can we use "" instead of it, and change it in ConfigUtil?

Then the code will be more simpler

Reply:
this is indeed better